lib/List.dbl

```diff
@@ -36,3 +161,95 @@ pub let iter2 {`re : {type X} -> Unit ->[|_] X} =
     | _                => `re ()
   end
   in iter
+
+pub let init (len : Int) f = 
+  let rec init_aux (i : Int) l f = if i > l then [] else (f i) :: init_aux (i+1) l f 
```

Comment:
Firstly, this line is too long. It should fit on 80 column [VT100](https://en.wikipedia.org/wiki/VT100) terminal.

Secondly, The `l` parameter of `init_aux` is not necessary. We can use `len` instead.